packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1825,26 +1825,11 @@ export const ActionsCantBeLoaded = {
  * @name Session Errors
  */
 // Session Errors
-/**
- * @docs
```

Comment:
Whatever we decide re: deleting vs deprecating errors when an experimental feature becomes stable, let's do it in all of these 5.7.0 PRs! (I left a similar comment in the `astro:config` PR.

Reply:
Noting that in the other PR, we were deleting, so I'm guessing that's fine here too, and resolving this!